fuzzing/tools/copy_corpus.py

```diff
@@ -0,0 +1,56 @@
+#
```

Comment:
Nit: I'm wondering if we should focus on the copying aspect of this operation in the name of the tool. How about calling this tool "make_corpus_dir"?